src/lib/yacc.rs

```diff
@@ -216,18 +239,23 @@ impl YaccParser {
                 }
 
             },
-            None => Err(YaccError{kind: YaccErrorKind::IllegalString, off: i})
+            None => {
+                Err(self.mk_error(YaccErrorKind::IllegalString, i))
+            }
         }
     }
 
-    fn parse_ws(&self, i: usize) -> YaccResult<usize> {
-        match RE_WHITESPACE.find(&self.src[i..]) {
-            Some((s, e)) => {
-                assert!(s == 0);
-                Ok(i + e)
+    fn parse_ws(&mut self, i: usize) -> YaccResult<usize> {
```

Comment:
I really don't want to treat `parse_ws` differently, especially as it might (in the future) report an error (e.g. maybe we interpret `\f` as an error).